<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDisplaySettings</class>
 <widget class="QDialog" name="DialogDisplaySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>334</width>
    <height>236</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>334</width>
    <height>236</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Display Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>18</y>
     <width>318</width>
     <height>105</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="displayVerticalLayout">
      <item>
       <widget class="QRadioButton" name="allNodesButton">
        <property name="text">
         <string>Display All Nodes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="showStructuresRadioButton">
        <property name="text">
         <string>Display Only Nodes in Structures</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="showSolutionsRadioButton">
        <property name="text">
         <string>Display Only Nodes in Solutions</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="nodeSubsetButton">
        <property name="text">
         <string>Display Nodes Subset...</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="buttonBoxVerticalLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="displaySettingsBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>128</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Node Subset Settings</string>
   </property>
   <widget class="QCheckBox" name="showIncompatibleCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>185</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Nodes with No Compatibilities</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>63</y>
      <width>325</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="displaySettingsGridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="praCheckBox">
       <property name="text">
        <string>Display Candidates in PRA Range</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="praSpinBoxRight">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="praSpinBoxLeft">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="showHoldCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>17</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Nodes on Hold</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogDisplaySettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogDisplaySettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodeSubsetButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>displaySettingsBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>praCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>praSpinBoxLeft</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>praCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>praSpinBoxRight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
